common/metrics/defs.go

```diff
@@ -1585,6 +1585,7 @@ const (
 	ConsistentQueryTimeoutCount
 
 	NumHistoryMetrics
+	DecisionTaskCompletedWithEmptyDecisionsCount
```

Comment:
plz change this to line 1477, just after `MultipleCompletionDecisionsCounter` and rename it `EmptyCompletionDecisionsCounter `

Reply:
could you plz also add the corresponding mapping at after https://github.com/uber/cadence/blob/a4561ec56beab9f4004327b92766b37a5ef66064/common/metrics/defs.go#L1789

Reply:
Done, however the list is not ordered alphabetically anyways.